src/components/displayResult.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class DisplayResult extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+      }
+    
+      render() {
+        const { total, next, operation } = this.props;
+        let display = 0;
+
+        if (!total && !next && !operation) {
+            display = 0;
+          }else if (!total && next) {
+            display = next;
+          } else if (operation && total && !next) {
+            display = `${total} ${operation}`;
+          } else if (total && next && operation) {
+            display = `${total} ${operation} ${next}`;
+          } else {
+              display = `${total}`;
+          }
+          return (
+           <div className="calculator-screen">{display}</div>
+           
+          );
+      }
+    }
+    
+    DisplayResult.propTypes = {
+      total: PropTypes.string,
```

Comment:
* Seems while working on the requested changes another error was created kindly correct the prop types for total it expects a string but gets a number instead 
![image](https://user-images.githubusercontent.com/52497246/129329639-5261ba29-f42d-407d-905f-594462f5e059.png)